source/upgrade.txt

```diff
@@ -99,6 +99,14 @@ Procedure
 
 #. Launch the fully-featured edition of |compass|.
 
+.. note::
+
+   If you are running macOS Catalina, or later versions, you may have to modify
```

Comment:
[nits] 

- I think we have too many commas here which makes the pace of reading this a little off.

- I don't think we need to say "later versions" we can just say "or later".

Suggest something like:

`If you are running macOS Catalina or later, you may...`